### DISCUSSION TITLE
More Robust Documentation

### DISCUSSION THREAD

**Discussion by wrobson-lllow** [payloadcms/payload#2993] [Feature Requests & Ideas]
I want to start out by saying I've really been enjoying using Payload for the last three months or so. It's definitely saved us a lot of time developing our current application. At the moment the one big thing that I've had issues with is the documentation. Often the examples or descriptions are boilerplate that work fine for showing syntax but do little for showing use cases. The entire documentation could use some more thought and work, but, there are a couple areas that seem particularly lacking. I see that the documentation for the rest API has been recently updated to include both request and response, which is many times more useful than it's previous version.
 
Would it be possible to update the Local API and the GraphQL API docs in the same manner as the REST API?

The other big areas that seem to need significant improvement are for custom react components and custom css. The newer version of the [ react docs](https://react.dev/) (as well as the docs for vue3) do a great job providing real use examples that are detailed enough to be useful for a variety of complex uses, where as the docs for payload often feel like they say "You can do X with payload!" instead of "This is how you do "X" with payload."

**Comment by zubricks** [comment]
Hey @wrobson-lllow we appreciate the candid feedback! We're always working on improving our docs, and these are all great ideas. I'll take this to the team and we'll keep all of this top of mind moving forward. Thanks again for the support!

**Reply by wrobson-lllow** [reply]
@zubricks  Much appreciated. I hope my request didn't come across too strong. As I said, I do really enjoy using payload.

**Reply by zubricks** [reply]
Not at all!
